.github/workflows/CI.yml

```
@@ -21,7 +21,7 @@ jobs:
           version: ${{ matrix.julia-version }}
 
       - name: Add package versions
-        run: julia --project=. -e 'using Pkg; pkg"dev .; add https://github.com/JuliaPlots/MakieRecipes.jl; add CairoMakie#master"'
+        run: julia --project=$(mktemp -d) -e 'using Pkg; pkg"dev .; add https://github.com/JuliaPlots/MakieRecipes.jl; add CairoMakie#master"'
```

This won't propagate down to the tests.  I'd recommend doing something like:
```suggestion
        run: julia --project=temp_for_test -e 'using Pkg; pkg"dev .; add https://github.com/JuliaPlots/MakieRecipes.jl; add CairoMakie#master"'

---

I was thinking that this might be a problem, but I don't get how the name change helps.

---

The name of the temp directory won't be accessible to the next stage, which will need that Project.toml

---

Also the fact that MakieRecipes is not yet registered might be a problem.
```julia
Testing RecipesPipeline
┌ Error: Pkg.Types.PkgError("expected package `MakieRecipes [3c562d8e]` to be registered")
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Operations.jl:1365
ERROR: expected package `MakieRecipes [3c562d8e]` to be registered
```